autogen/agentchat/contrib/vectordb/chromadb.py

```diff
@@ -24,15 +24,15 @@ class ChromaVectorDB(VectorDB):
     """
 
     def __init__(
-        self, *, client=None, path: str = None, embedding_function: Callable = None, metadata: dict = None, **kwargs
+        self, *, client=None, path: str = "tmp/db", embedding_function: Callable = None, metadata: dict = None, **kwargs
     ) -> None:
         """
         Initialize the vector database.
 
         Args:
             client: chromadb.Client | The client object of the vector database. Default is None.
                 If provided, it will use the client object directly and ignore other arguments.
-            path: str | The path to the vector database. Default is None.
+            path: str | The path to the vector database. Default is `tmp/db`.
```

Comment:
Is this change worth more documentation, such as "the default was ... for version ... "?

Reply:
This change is based on PR #2263 which has not been released. What about adding `the default was ...` if #2263 and this PR are not in the same release version?

Reply:
Updated.